byte relay_number;
bool relay_request;
int32 relay_off_delay;
byte io_number;
bool io_request;
---
bool status;
